whitepapers/0007_governor.md

```diff
@@ -4,37 +4,88 @@
+* An attacker could utilize liquidity pools and other bridges to launder illicitly minted wrapped assets.
+
+## FAQ
+- **Who is the Governor?**
+    - The Governor is not a person, but a software feature built into the guardian reference implementation.
+- **When does the Governor step in?**
+    - The Governor delays messages for up to 24 hours if the aggregate value of messages originating from one chain in the past 24 hours exceeds a threshold that is set per chain, which happens very rarely.
+- **In what direction are Governor thresholds for delays imposed?**
+    - Transfer volume is calculated *from* each chain.
+- **Are Governor thresholds per wallet?**
+    - Governor thresholds are imposed globally from each chain, to address [Sybil-attacks](https://en.wikipedia.org/wiki/Sybil_attack), in which an attacker could split a large transfer into many small transfers.
+- **How can the Governor help protect Wormhole users?**
+    - The Governor can help reduce the impact of a hack or fork by giving the Guardians additional time to respond to any extraordinary events. E.g., if a chain forks, while the Governor and the new fork delay a Wormhole message, do not include that Wormhole message anymore, the Guardians have time to decide which fork to adopt and whether or not to confirm the pending messages, to ensure that Wormhole messages represent the state of the origin chain as accurately as possible.
+- **How does the Governor *exactly* decide which messages to confirm immediately and which to delay?**
+    - The Governor has two configurable values for each chain: The 24h threshold and the single-transaction threshold. These two thresholds only apply to messages originating **from** a given chain. Messages are processed differently if their value exceeds the single-transaction threshold or not. We call these small transfers and large transfers. Large transfers will always be delayed for 24 hours. Small transfers will be confirmed immediately unless the sum of all small transfers in the last 24 hours, plus the transfer in question, exceeds the 24h-threshold (this is to protect against an attacker splitting a large transfer into many small transfers). Those small transfers will be delayed as long as the 24h threshold is exceeded, but never for longer than 24 hours. If a small transfer would still cause the 24h threshold to be exceeded after 24 hours (e.g., because there was a constant flow of smaller transfers), it will be confirmed regardless and not count towards the 24h-threshold.
+- **Is it possible to have my transfer delayed by Governor?**
+    - The above answer explains that transactions can be delayed during high utilization times.  However, the maximum possible wait time is 24hrs, though most transactions have no delay.
```

Comment:
Is it worth mentioning that the guardian operator can manually extend the 24 hour timer if they need more time to investigate?